src/analytics/segment.js

```diff
@@ -39,6 +40,7 @@ const extraProperties = store => {
     platformVersion: Platform.Version,
     sessionId,
     devicesCount: devices.length,
+    devicesVersions,
```

Comment:
segment DO NOT support arrays of data. this would badly break and never track any analytics if i recall correctly.

in any case, we need to follow the same impleemntation as the one we will do on LLD where i had some proposal of changes (see other PR on LLD)

Reply:
ha ok, i looked quickly and I saw "The Schema will show you how recently we’ve seen your event properties and their type (ex: array, boolean, data, number, text)"

Reply:
yeah i think in practice it is not properly integrated (maybe due to the "destinations" we use)